package.json

```diff
@@ -50,13 +65,12 @@
     "ramda": "^0.25.0"
   },
   "devDependencies": {
-    "eslint": "^4.19.1",
-    "eslint-config-twine": "github:TwinePlatform/eslint-config-twine",
-    "eslint-plugin-import": "^2.12.0",
-    "eslint-plugin-jsx-a11y": "^6.0.3",
-    "eslint-plugin-react": "^7.9.1",
+    "@fimbul/bifrost": "^0.9.0",
```

Comment:
what's this for?

Reply:
It's a tslint dependency, I'll stick it in the other repo